### DISCUSSION TITLE
Ignore 'vendor' directory

### DISCUSSION THREAD

**Discussion by Mamaduka** [spatie/wordpress-ray#3] [Ideas]
I believe it's there for the sake of WP.org deployments.

We can probably remove it and let deploy to WP workflow handle `composer install`. This can be done by replacing npm [build-script](https://github.com/spatie/wordpress-ray/blob/master/.github/workflows/deploy-to-wordpress.yml#L14) with composer one.

**Comment by freekmurze** [comment]
Thanks for the suggestion, feel free to submit a PR that improves this 👍
